performance-es.md

```diff
@@ -0,0 +1,22 @@
+# Escribir y optimizar código en Go
+
+Este documento describe las mejores prácticas para escribir código en Go de alto rendimiento.
+
+Si bien se discutirán maneras de optimizar servicios individuales (almacenamiento en caché, etc.), el diseño de sistemas distribuidos de alto rendimiento está fuera del alcance de este trabajo. Ya hay buenos textos sobre monitorización y diseño de sistemas distribuidos. Este tema abarca un conjunto completamente diferente de investigación y concesiones en el diseño.
```

Comment:
I also have some doubts about this one, since as I understood the original text, the last sentence refers to the distributed systems topic, not the text or article in itself. Does this make sense? Maybe it needs a rephrase to avoid confusion though.

Reply:
I actually think `tema` refers to the  `diseño de sistemas distribuidos`. I think that last sentence is still reffering to the subject that will not be covered in this document.

My suggestion would be: `s/Este tema/Dicho tema` which translates to `Said topic`.

Reply:
I like this one more @rocajuanma :+1:

Reply:
s/Ya ha hay buenos textos
Actualmente existen textos concretamente detallados sobre ...

Reply:
Yes, the last sentence refers to the fact that optimizing distributed systems is completely different from optimizing for CPU and memory, which is the main focus of perfbook.

I'll see if I can clarify the original English.

Reply:
Given the actual meaning in English I like @rocajuanma suggestions better, makes total sense.